components/project-card/project-card-detailed.jsx

```diff
@@ -91,21 +93,54 @@ class DetailedProjectCard extends React.Component {
     });
   }
 
+  setSocialPanelHeight() {
+    let article = document.querySelector('.main-content');
```

Comment:
Same as my comment in another PR. We should avoid using `document.querySelector` in React unless it's something we can access it via `ref`. 

Bad scenario could be:

There could be another `.main-content` rendered on the page (outside of this component) and by using `document.querySelector('.main-content')` we could be targeting the wrong element.

Reply:
I added `ref`s! 😁